rules/autoconsent/justwatch-com.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "justwatch.com",
+  "cosmetic": true,
+  "prehideSelectors": [".consent-banner"],
+  "detectCmp": [{ "exists": ".consent-banner" }],
```

Comment:
This triggers on twitch.tv (false positive). We should separate opt-out rule for that. Can we use a url pattern to limit this. It also seems like there are options to opt-out available via the 'more info' link when accessing from the US, so we could make this a proper rule.

Reply:
Opt-out rule created instead of the notice banner one. Ready for review!